tests/json/schema_0_properties/input.py

```diff
@@ -114,9 +114,9 @@
 
 psi4.compare_integers(True, json_data["success"], "JSON Success")                           #TEST
 for k in expected_return_result.keys():                                                     #TEST
-    psi4.compare_arrays(expected_return_result[k], json_data["return_result"][k], 6, "Result: " + k.upper())  #TEST
+    psi4.compare_arrays(expected_return_result[k], json_data["return_result"][k], 5, "Result: " + k.upper())  #TEST
```

Comment:
Should we change this to 5 for all schema tests as the tolerances are not touched in any of them?

Reply:
Done